pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -346,13 +323,19 @@ func getMembersByPoolID(client *gophercloud.ServiceClient, id string) ([]v2pools
 			return false, err
 		}
 		members = append(members, membersList...)
-
+		if len(members) > 1 {
```

Comment:
This particular function is intended to return multiple results - see pluralised name and array return type.